_help.sh

```diff
@@ -0,0 +1,25 @@
+GOGO_ROOT=${GOPATH}/src/github.com/gogo/protobuf
```

Comment:
Why name it `_help.sh`? Is `help.sh` OK?

Reply:
it is general convention in bash. Utility stuff always starts with prefix `_`.

Reply:
Got it!